esy-build-package/Build.re

```diff
@@ -358,11 +358,27 @@ let commitBuildToStore = (config: Config.t, build: build) => {
         empty
         % config.fastreplacestringCmd
         % p(path)
-        % p(origPrefix)
-        % p(destPrefix)
+        % origPrefix
+        % destPrefix
       );
     Bos.OS.Cmd.run(cmd);
   };
+  let rewritePrefixesInFile = (~origPrefix, ~destPrefix, path) => {
+    let origPrefixString = Path.to_string(origPrefix);
+    let destPrefixString = Path.to_string(destPrefix);
+    switch (System.Platform.host) {
+    | Windows =>
+        /* On Windows, the slashes could be either `/` or windows-style `\` */
+        /* We'll replace both styles */
+        let%bind () = rewritePrefixInFile(~origPrefix=origPrefixString, ~destPrefix=destPrefixString, path);
+        let normalizedOrigPrefix = Path.normalizePathSlashes(origPrefixString);
+        let normalizedDestPrefix = Path.normalizePathSlashes(destPrefixString);
```

Comment:
I wonder if we should always rewrite to the normalized form.
So:
- `non-normalized-input -> normalized-output`
- `normalized-input -> normalized-output`

Probably wouldn't hurt.

Reply:
(As long as `Path.normalizePathSlashes` never ever changes the length in bytes (because binary files can't be rewritten to different lengths).